SPEC.md

```diff
@@ -266,6 +266,26 @@ line of the file.
 
 Offsets, however are 0-based. A Position of `{ "offset": 4 }` represents the _fifth_ character in the file. Importantly, the `offset` is from the beginning of the file, not the beginning of a line. Newline characters (and all characters) count when computing an offset.
 
+### Other locations
+
+Other locations is an optional array of location objects
+
+```json
```

Comment:
Personally I don't think a full example here is informative or necessary. It's just an array of an object type described elsewhere, and the full `Issue` schema makes that clear.

If you think it's important to keep, though, please make the formatting consistent with other cases that describe a top-level key, e.g. the example in the `Trace` section. (Specifically, I'm referring to *not* having the wrapping `{}` around the whole thing, which make it seem like this whole snippet is a valid object, which it isn't in our schema.)